src/Services/QueuedJobLogger.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symbiote\QueuedJobs\Services;
+
+use Psr\Log\AbstractLogger;
+use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
+
+/**
+ * Writes log output to a job descriptor
+ */
+class QueuedJobLogger extends AbstractLogger
+{
```

Comment:
```suggestion
{
    use Injectable;
```

To allow Injector to instantiate